Zapper-fi/studio
# feat(vendor-finance): Adds support for v2 ETH pools + swaps subgraph endpoints

## Description
This PR contains two non-insignificant changes.
1. Adds support for v2 pools on Ethereum
2. Swaps subgraph URLs in favor of custom entity fetcher URL for all networks.


## Checklist

- [x] (optional) As a contributor, my Twitter handle is: @BossMcBara

## How to test?
To see that the new entity fetcher URL works follow the below:
1. Run `pnpm dev`
2. Verify V2 on Ethereum: `http://localhost:5001/apps/vendor-finance/balances?addresses%5B%5D=0xC6c48E582405BE32f25EdBa211CCaa820058Cb2f&network=ethereum`
3. Verify V2 on Arbitrum: `http://localhost:5001/apps/vendor-finance/balances?addresses%5B%5D=0x2a6334c30f36d1eefd64689c590dd199d3475972&network=arbitrum`
4. Verify v1 on Ethereum: `http://localhost:5001/apps/vendor-finance/balances?addresses%5B%5D=0x245cc372C84B3645Bf0Ffe6538620B04a217988B&network=ethereum`
5. Verify v1 on Arbitrum: `http://localhost:5001/apps/vendor-finance/balances?addresses%5B%5D=0xD6587a974C7D3ecE23Fa53d5606da6B291311F6f&network=arbitrum`
